.ki-scripts/README.md

```diff
@@ -0,0 +1,45 @@
+# Kitty Items Scripts
+
+## Version Check
+
+A script that compares the current process's Node version against the [package.json](/package.json) Node engine requirement.
+
+## Startup
+
+This script is called when deploying both local & testnet versions of the Kitty Items application. Multiple processes are executed to initialize both the database as well as an admin service account required for the application to function.
+
+### Begin Process
+The script begins by ending all running processes from previously launched applications.
+
+### Check for correct Node version
+In this step, the script checks the process's Node version against the Node engine requirement in the root [package.json](/package.json). Similar to [version-check.js](/.ki-scripts/version-check.js).
```

Comment:
Why is this done twice?

Reply:
the stand-alone version check script is ran through `npm run preinstall` 
the logic is also in the startup script. adding a line to the `Version Check` description on how to run the preinstall script